lib/handleRequest.js

```diff
@@ -54,9 +50,9 @@ function handleRequest (req, res, params, context) {
 }
 
 function handler (reply) {
-  var valid = validateSchema(reply.context, reply.request)
-  if (valid !== true) {
-    reply.code(400).send(wrapValidationError(valid))
+  var result = validateSchema(reply.context, reply.request)
+  if (result) {
```

Comment:
This could be a performance penalty. Could you make the previous check?

Reply:
Agree, instead of let JavaScript perform the boolean check, write directly the type you expect to see here :)